modules/text/src/precomp.hpp

```diff
@@ -48,6 +48,9 @@
 #include "text_config.hpp"
 
 #ifdef HAVE_TESSERACT
+#if !defined(USE_STD_NAMESPACE)
+#define USE_STD_NAMESPACE
```

Comment:
There are no mentions of this macro in the Opencv code? Why do you need this?

Reply:
@LaurentBerger  Thanks man, your code worked like a charm.